src/Kentico.Xperience.Lucene.Admin/LuceneConfigurationModel.cs

```diff
@@ -19,55 +29,100 @@ public class LuceneConfigurationModel
     public IEnumerable<string> ReusableContentTypeNames { get; set; } = Enumerable.Empty<string>();
 
+
+    /// <summary>
+    /// The languages that the index will support.
+    /// </summary>
     [GeneralSelectorComponent(dataProviderType: typeof(LanguageOptionsProvider), Label = "Indexed Languages", Order = 4)]
     public IEnumerable<string> LanguageNames { get; set; } = Enumerable.Empty<string>();
 
-    [DropDownComponent(Label = "Channel Name", DataProviderType = typeof(ChannelOptionsProvider), Order = 5)]
-    public string ChannelName { get; set; } = "";
 
+    /// <summary>
+    /// The name of the indexing strategy to be used for determining how and when items are indexed.
+    /// </summary>
     [DropDownComponent(Label = "Indexing Strategy", DataProviderType = typeof(IndexingStrategyOptionsProvider), Order = 6)]
     public string StrategyName { get; set; } = "";
 
+
+    /// <summary>
+    /// The name of the Lucene analyzer to be used for text analysis during indexing and searching.
+    /// </summary>
     [DropDownComponent(Label = "Lucene Analyzer", DataProviderType = typeof(AnalyzerOptionsProvider), Order = 7)]
     public string AnalyzerName { get; set; } = "";
 
+
+    /// <summary>
+    /// The rebuild hook of the index.
+    /// </summary>
     [TextInputComponent(Label = "Rebuild Hook", Order = 8)]
     public string RebuildHook { get; set; } = "";
 
+
+    /// <summary>
+    /// An empty constructor for model binding.
+    /// </summary>
     public LuceneConfigurationModel() { }
 
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="LuceneConfigurationModel"/> class based on an existing <see cref="LuceneIndexModel"/>.
+    /// </summary>
+    /// <returns><see cref="LuceneConfigurationModel"/></returns>
     public LuceneConfigurationModel(
         LuceneIndexModel luceneModel
     )
     {
         Id = luceneModel.Id;
         IndexName = luceneModel.IndexName;
         LanguageNames = luceneModel.LanguageNames;
-        ChannelName = luceneModel.ChannelName;
         StrategyName = luceneModel.StrategyName;
         AnalyzerName = luceneModel.AnalyzerName;
         RebuildHook = luceneModel.RebuildHook;
+#pragma warning disable CS0618 // Type or member is obsolete
         Paths = luceneModel.Paths;
+        ChannelName = luceneModel.ChannelName;
+#pragma warning restore CS0618 // Type or member is obsolete
+        Channels = luceneModel.Channels;
         ReusableContentTypeNames = luceneModel.ReusableContentTypeNames;
     }
 
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="LuceneIndexModel"/> class based on the current configuration.
+    /// </summary>
+    /// <returns><see cref="LuceneIndexModel"/>.</returns>
     public LuceneIndexModel ToLuceneModel() =>
         new()
         {
             Id = Id,
             IndexName = IndexName,
             LanguageNames = LanguageNames,
```

Comment:
Paths and channel name must assigned just like before to have public API operational as before.